ci/macos/prepare.sh

```diff
@@ -9,7 +9,12 @@ brew update
 brew upgrade cmake
 brew install openssl@1.1
 
-sudo python3 -m pip install btest
+# Install btest in a venv since under macos pip refuses to install into the
+# global prefix since it is explicitly "externally managed".
+python3 -mvenv ./btest.venv
```

Comment:
In the Zeek repo we use `pip3 install --break-system-packages` as a workaround here and there, would that work?

But not an overly strong opinion.

Reply:
That would certainly simplify things and wouldn't require us to encode knowledge about some "magic directory" `btest.venv` in multiple places. To me personally ` --break-system-packages` looks like a yucky non-solution though.

Reply:
I am merging this with the venv, in particular to avoid people copying this (this is less an issue for us on ephemeral CI machines).